tests/test.cli.js

```diff
@@ -62,13 +46,6 @@ describe('Basic CLI tests', function() {
       'Not enough non-option arguments'));
   });
 
-  it('should show error if incorrect type', () => {
```

Comment:
Makes more sense to remove than to skip?

I wonder about even just commenting things out with a TODO that says "nix these if we still aren't using them by v1" or something; technically less git magic but it seems easier to manage in my brain.